blog/2023/05-14-android-v250/index.mdx

```diff
@@ -0,0 +1,80 @@
+The player interface itself now has a back button to close the player and a new playback option was added to pick between hardware and software decoding, if the default hardware decoder causes issues for the currently played video.
+
+In the client settings (which you should make sure to check out) we added the option to pick the initial orientation for video playback and now allow choosing whether you want video with SSA subtitles to be transcoded (for full styling support) or not (to direct play with limited styling). There's also a link to the operating system's subtitle customization settings, which the integrated player uses to style your subtitles. Note that the subtitle settings in the Jellyfin settings only apply to the web player due to technical reasons.
+
+## Other Tweaks
+
+Thanks to @susonthapa, the app now shows a splash screen with the Jellyfin logo on start, instead of the previous, boring gray screen where you didn't see whether the app was actually doing anything. While the app is connecting, there's now a cancel button so that you can quickly jump to the connect screen to select a different server if the current one isn't available at the moment.
+The connect screen itself was reworked and rewritten in JetPack Compose, a new UI framework by Google that allows fast and easy UI development. We plan to use it for more parts of the app in the future, especially for the long-term goal of a completely native app experience.
+We also added a monochrome icon so that Jellyfin will look great on your home screen on supported launchers on Android 13 and above.
+
+## TODO – To mention or not to mention?
+
+- Cronet - very technical and most probably don't care. The percentage of users that deployed QUIC is probably also quite low
+- PGSSUB - a full sentence for just one subtitle codec? Idk..
+- Disabled Safe browsing - only applies to very few users and might confuse/scare others
+
+## Download Now
+
+<a className="margin-right--md" href="https://play.google.com/store/apps/details?id=org.jellyfin.mobile">
+  <img width="153" alt="Jellyfin on Google Play" src="/images/store-icons/google-play.png" />
+</a>
+
+<a className="margin-right--md" href="https://www.amazon.com/gp/product/B081RFTTQ9">
+  <img width="153" alt="Jellyfin on Amazon App Store" src="/images/store-icons/amazon.png" />
+</a>
+
+<a href="https://f-droid.org/en/packages/org.jellyfin.mobile/">
+  <img width="153" alt="Jellyfin on F-Droid" src="/images/store-icons/fdroid.png" />
+</a>
+
+Direct downloads are always available from [our repository](https://repo.jellyfin.org/releases/client/android/).
+
+## Full Release Notes
+
+The full (technical) release notes are available on [GitHub](https://github.com/jellyfin/jellyfin-android/releases/tag/v2.5.0).
```

Comment:
```suggestion
The complete list of changes for this release, including all merged pull requests, can be found on
[GitHub](https://github.com/jellyfin/jellyfin-android/releases/tag/v2.5.0).
```

Reply:
I'm not sure how to feel about the “including”, it *is* just the pull requests..

Reply:
How about

```suggestion
The complete list of changes for this release with all merged pull requests can be found on
[GitHub](https://github.com/jellyfin/jellyfin-android/releases/tag/v2.5.0).
```

Reply:
Better.